package com.kodilla.exception.test;

public class SecondChallengeException extends Exception {
}
